Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/OverlaySheetService.java

```diff
@@ -211,7 +211,7 @@ public void onViewHolderBind(SheetAdapter.SheetViewHolder holder, int position,
             }
             updateTipView(holder, model);
 
-            if (key != null && key.equals("yatrisathiprovider")) {
+            if (key != null && (key.equals("yatrisathiprovider") || key.equals("yatripartner"))) {
```

Comment:
replace yatripartner with yatriprovider as the service also will be yatriprovider

Reply:
done